### ISSUE TITLE
Address security vulnerabilities

### ISSUE THREAD

**Issue by ALRubinger** [TBD54566975/incubating-web5-labs#50]
On #48:

```
$> npm install
npm WARN deprecated multibase@4.0.6: This module has been superseded by the multiformats module

added 428 packages, and audited 429 packages in 11s

98 packages are looking for funding
  run `npm fund` for details

1 high severity vulnerability
```

**Comment by ALRubinger** [comment]
Pushed `issue-47/update-sdk` branch to `upstream`, incorporating #48 from @bobbilee19 and a fix for the security vulnerability here in b3fcd78.

**Comment by ALRubinger** [comment]
Se can close when Draft PR #55, which contains this fix, is pushed `upstream` to `main`
